src/test/java/com/stripe/functional/ChargeTest.java

```diff
@@ -148,6 +151,14 @@ public void testMarkSafeWithTypedParams() throws StripeException {
         fraudDetailsUntypedParam("safe"));
   }
 
+  @Test
+  public void testSearch() throws Exception {
+    ChargeSearchResult result =
+        Charge.search(ChargeSearchParams.builder().setQuery("currency=USD").build());
```

Comment:
Nit: Let's do `currency:"USD"`. This isn't a valid query as is and you never know if someone will use his test to understand how to use the API.